enterprise/micronaut/src/org/netbeans/modules/micronaut/db/MicronautRepository.java

```diff
@@ -132,6 +139,14 @@ protected List<FileObject> createFromTemplate(CreateDescriptor desc) throws IOEx
                         DialogDisplayer.getDefault().notifyLater(new NotifyDescriptor.Message(Bundle.MSG_NoEntities(sourceGroup.getRootFolder().getPath()), NotifyDescriptor.ERROR_MESSAGE));
                         return Collections.emptyList();
                     }
+                    List<Item> repoType = new ArrayList<>();
+                    Item pageableItem = new Item(Bundle.MSG_PageableRepository(), null /*Bundle.DESC_PageableRepository()*/);
+                    repoType.add(new Item(Bundle.MSG_CrudRepository(), null /*Bundle.DESC_CrudRepository()*/));
```

Comment:
Why is the item's  description commented out?

Reply:
It was there initially, but Jirka suggested to remove it, since it is too long and does not fit well in one line. We can use it as 'detail' string when `Item` is extended to support it - it currently supports name and description of [QuickPickItem](https://code.visualstudio.com/api/references/vscode-api#QuickPickItem).